tools/serverpod_cli/lib/src/generator/types.dart

```diff
@@ -454,6 +454,8 @@ class TypeDefinition {
         return DefaultValueAllowedType.double;
       case ValueType.string:
         return DefaultValueAllowedType.string;
+      case ValueType.uuidValue:
```

Comment:
Comment: This is part of the second commit but would have been required to be able to run the tests in the first commit.

Reply:
Yes, you are right, i missed this when i was working on the commit history, sorry!